src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ServiceProviderComment.java

```diff
@@ -0,0 +1,59 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+/**
+ * Example FTX+SPC+++red blood cell seen, Note low platelets'
+ */
+@RequiredArgsConstructor
+@Getter
+@EqualsAndHashCode(callSuper = false)
+public class ServiceProviderComment extends Segment {
```

Comment:
I just took it from the qualifier description in the LSR_04_A_001. My logic to use the qualifier description as the class name was because the similar segment with the same qualifier is also used by the SG18 and it has the same qualifier description in the table, so that's why I gave to the class such a name. However, I might be wrong. Do you think it will be better to call it something like ServiceProviderFreeText?

<img width="905" alt="Screenshot 2021-02-04 at 09 29 31" src="https://user-images.githubusercontent.com/36967438/106872763-93437d00-66cb-11eb-8a28-238aeb298525.png">

Reply:
For consistency, I'd like either this to be called `ServiceProviderCommentFreeText`, or to rename the other one to just `ReferencePopulationDefinition`. I have a slight preference for the former option, but would accept both.

Reply:
We definitely want to be consistent. Thanks! b9721f3ac23ef511095b169e3996c1f1e959f62c